src/utils/validation/EmployeeArcInfoValidation.js

```diff
@@ -0,0 +1,78 @@
+import * as yup from 'yup';
+import { requiredFieldError } from '../constants';
+
+function formatDate(date) {
+  return new Date(date).toLocaleDateString().replaceAll('/', '.');
+}
+
+export const EmployeeArcInfoValidationSchema = yup.object().shape({
```

Comment:
```suggestion
export const EmployeeArchiveInfoValidationSchema = yup.object().shape({
```
здесь Arc, а в другом месте было Arh, поэтому надо писать полностью, тогда не надо будет запоминать где какое сокращение